src/buffer/out/OutputCellIterator.cpp

```diff
@@ -518,8 +518,7 @@ OutputCellView OutputCellIterator::s_GenerateView(const CHAR_INFO& charInfo) noe
         dbcsAttr.SetTrailing();
     }
 
-    TextAttribute textAttr;
-    textAttr.SetFromLegacy(charInfo.Attributes);
+    const TextAttribute textAttr(charInfo.Attributes);

```

Comment:
tiny nit: it took me a while looking at this to get over the "most vexing parse"; can we use `{}` universal init syntax?

Reply:
Yeah I'm a fan of the `{}` syntax too, but most of the surrounding code is using `()` initialization, at least in the areas I was modifying. So if we want to change that, I think it's best to handle that in a separate code cleanup PR that replaces everything consistently.